src/lib/utils.js

```diff
@@ -70,14 +70,16 @@ export const createDeepLinkUrl = ({ type, callbackUrl, ...params }) => {
   Object.entries(params)
     .filter(([, value]) => ![undefined, null].includes(value))
     .forEach(([name, value]) => url.searchParams.set(name, value));
+  console.log('params', params);
+  console.log('url', url);
   return url;
 };
 
 export const createOnAccountObject = (address) => ({
-  address: () => address,
-  sign: () => {
-    throw new Error('Private key is not available');
-  },
+  address,
+  signTransaction: () => null
```

Comment:
is this supposed to be that way?

Reply:
it's used only in the mobile mode, where the signing is done later through the URL callback